src/uk/gov/defra/ffc/Cleanup.groovy

```diff
@@ -0,0 +1,21 @@
+package uk.gov.defra.ffc
+
+import uk.gov.defra.ffc.Helm
+
+class Cleanup implements Serializable {
+  static def kubernetesResources(ctx, environment, repoName, branchName) {
```

Comment:
This method was called `cleanup` as most of the logic is related to determining the repo and branch to cleanup, with only one line refering to kubernetes - `Helm.undeployChart(ctx, environment, repoName, "pr$closedPrNo")`.
It's likely that other resources will be removed by this method such as ephemeral environments, so should have a more generic name.

A better name would be `cleanupPrResources` or similar.

Reply:
Renamed to `prResources` so it can be called as `Cleanup.prResources`